pkg/template/release.go

```diff
@@ -16,7 +16,7 @@ func BuildImages(registryDomain string, versions Versions) Images {
 		CalicoKubeControllers:        buildImage(registryDomain, "giantswarm/kube-controllers", versions.Calico, ""),
 		CalicoNode:                   buildImage(registryDomain, "giantswarm/node", versions.Calico, ""),
 		Etcd:                         buildImage(registryDomain, "giantswarm/etcd", versions.Etcd, ""),
-		Hyperkube:                    buildImage(registryDomain, "giantswarm/hyperkube", versions.Kubernetes, ""),
+		Hyperkube:                    buildImage(registryDomain, "giantswarm/hyperkube", strings.TrimPrefix(versions.Kubernetes, "v"), ""),
```

Comment:
The new images we're building don't have a "v" prefix: https://quay.io/repository/giantswarm/hyperkube?tab=tags

Reply:
I know a little late, but can we add the `v` on the images? :P

Reply:
We're building the image with architect-orb which doesn't seem to allow adding a prefix. We'd have to do it for all of our images I think.

Reply:
Okay . fine for now then :)